vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/impl/JcrPackageDefinitionImpl.java

```diff
@@ -646,6 +646,13 @@ public String getCreatedBy() {
         return get(PN_CREATED_BY);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public String getGenerator() {
```

Comment:
Can you add `@Override`

Reply:
Not convinced. I agree that "@Override" is better, but shouldn't we do this consistently here? Any way, your choice, just let me know.